.github/labeler.yml

```
@@ -0,0 +1,17 @@
+frontend:
+  - changed-files:
+      - any-glob-to-any-file: 'src/frontend/*'
+
+documentation:
+  - changed-files:
+      - any-glob-to-any-file: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)